ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/atomicaction/AtomicActionAsyncTest.java

```diff
@@ -49,6 +56,28 @@ public void testAbort() throws Exception {
     }
 
     @Test
+    public void testAbortAsyncAC() throws Exception {
```

Comment:
you are right, the true/ActionStatus.COMMITTED I was relying on the behaviour in the existing test to be correct which it is not. I have improved this test and corrected the previous test